.env.example

```diff
@@ -28,4 +28,16 @@ DATABASE_HOST=postgres
 REDIS_URL=redis://redis:6379
 
 # Auth scheme
-AUTH_SCHEME=fxa
\ No newline at end of file
+AUTH_SCHEME=fxa
+
+IMAP_HOST=localhost
+IMAP_PORT=993
+IMAP_TLS=True
+
+SMTP_HOST=localhost
+SMTP_PORT=465
+SMTP_TLS=True
+
+JMAP_HOST=localhost
+JMAP_PORT=7768
```

Comment:
I just took the first number of the ord value for each letter of jmap. This is technically an http listener.